### PR TITLE
SCMOD-9686: Updated Release Notes

### PR DESCRIPTION
**Pull request by rahul-kulkarni73**

### CODE REVIEW

File: release-notes-5.0.0.md [hunk 1]
```diff
@@ -4,7 +4,8 @@
 ${version-number}
 
 #### New Features
-- **SCMOD-8516**: Extend the security hardening of Java base images by disabling TLS algorithms mentioned [here](https://github.com/CAFapi/opensuse-java8-images/blob/develop/src/main/docker/disableWeakTlsAlgorithms.patch)
+- **SCMOD-8516**: Extend the security hardening of Java base images by disabling TLS algorithms mentioned [here](https://github.com/CAFapi/opensuse-java8-images/blob/develop/src/main/docker/disableWeakTlsAlgorithms.patch)  
+- **SCMOD-9686**: Included additional classes from tomcat-juli.jar. Added additional IDE files to gitignore.
```
Comment (michael-bryson) [inline release-notes-5.0.0.md]: Why is the gitignore changes being called out, we stated in the standup and the JIRA ticket that those changes should be reverted?

Comment (rahul-kulkarni73) [reply release-notes-5.0.0.md]: Done

File: release-notes-5.0.0.md [hunk 2]
```diff
@@ -4,7 +4,8 @@
 ${version-number}
 
 #### New Features
-- **SCMOD-8516**: Extend the security hardening of Java base images by disabling TLS algorithms mentioned [here](https://github.com/CAFapi/opensuse-java8-images/blob/develop/src/main/docker/disableWeakTlsAlgorithms.patch)
+- **SCMOD-8516**: Extend the security hardening of Java base images by disabling TLS algorithms mentioned [here](https://github.com/CAFapi/opensuse-java8-images/blob/develop/src/main/docker/disableWeakTlsAlgorithms.patch)  
+- **SCMOD-9686**: Included additional classes from tomcat-juli.jar.
```
Comment (michael-bryson) [inline release-notes-5.0.0.md]: Could you list the additional classes added here please?

**Review by michael-bryson — APPROVED** [review]
Approving on the basis that the additional classes are listed in the release notes.

File: release-notes-5.0.0.md (hunk 2 above)
Comment (rahul-kulkarni73) [reply release-notes-5.0.0.md]: done.

**Review by michael-bryson — CHANGES_REQUESTED** [review]
Sorry but the format does not look very good, see [here](https://github.com/CAFapi/opensuse-tomcat-image/blob/e84d77203bf6cedb0b54e9da510bb965d7602bcb/release-notes-5.0.0.md)

Maybe put them in a bulleted list and how about providing links to the apache documentation for these to give consumers some context. This is an open source repo so even more important for these things to look right.

Links for example:
- https://tomcat.apache.org/tomcat-9.0-doc/api/org/apache/juli/FileHandler.html
- https://tomcat.apache.org/tomcat-9.0-doc/api/org/apache/juli/AsyncFileHandler.html

**Review by michael-bryson — APPROVED** [review]

**Pull request merged**